### DISCUSSION TITLE
Garage Accessory Config Digital Output Orders

### DISCUSSION THREAD

**Discussion by ta2p** [RavenSystem/esp-homekit-devices#1651] [MEPLHAA Scripts]
The outputs do not execute more than two orders. Only runs the first two.
Like in this, (12-relay, 13-led, 0-button)
{"c":{"o":0,"l":13},"a":[{"0":{"r":[{"g":12,"v":1,"i":1}]},"1":{"r":[{"g":12,"v":1,"i":1},{"g":13,"v":1,"i":28},{"g":12,"v":1,"i":1}]},"t":40,"s":1,"d":28,"b":[{"g":0,"t":1}]}]}

**Comment by RavenSystem** [comment]
Third order is the same of first order: `{"g":12,"v":1,"i":1}`
